.ci.yaml

```diff
@@ -4143,7 +4138,6 @@ targets:
         {
           "sdk_version": "14c18"
         }
-    bringup: true
 
   - name: Mac_ios native_assets_ios_simulator
```

Comment:
Temporarily moved Xcode debug tests to presubmit, so I could confirm they still pass. Will move back around PR is reviewed